### DISCUSSION TITLE
Backport of a bug fix to an older release

### DISCUSSION THREAD

**Discussion by newlinefu** [maplibre/maplibre-gl-js#2618] [Q&A] [answered]
Good afternoon.
In our project, we use maplibre with version 1.5.2. Recently, there was a need to fix the https://github.com/maplibre/maplibre-gl-js/pull/2479 bug, the changes made in version 3.0.0. After some discussion, it was decided to change version in our project later, but we need a fix soon. 
Can you please tell me the best way for me to issue a fix in 1.x.x, if at all possible.

**Comment by HarelM** [comment, marked as answer]
No one had done this for a long time, so I'm not sure I have a good idea how to do that.
I would start with opening a PR against the 1.x branch that both fixes this issue and bumps the version.
After that is merged we can try and see if the publishing of a version will work for this branch or not, but I can't commit to anything.
If this doesn't work you'll need to create a PR to fix this.
Having said all that, I think you might be better off with using `patch-package` npm package to add a patch in your local deployment.
Once you migrate to 3.0 you could simply delete the patch file.
cc @wipfli I think you were the last one to attempt this, didn't you?

**Reply by newlinefu** [reply]
Thanks for the answer. I will try the solution with patch-package, but I'm not quite sure if it will suit us yet

**Reply by wipfli** [reply]
I think it was someone else.
